msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -53,38 +78,154 @@ See the following video to learn how to embed Share to Teams button:
+```
+
+All share buttons are rendered on the page. If an optional `options` object is supplied with a list of elements, those elements are rendered into share buttons.
+
+---
+
+The following are the launcher.js definitions:
+
+| Property | HTML attribute | Type | Required | Default | Description |
+| -------------- | ---------------------- | --------------------- | ------- |  ------- |---------------------------------------------------------------------- |
+| url | `data-href` | String | Yes | NA | URL that can be used to pin the content to an upcoming meeting or to share in an ongoing meeting. Required if allow-share-in-meeting is set as true. |
+| appId | `data-app-id` | String | Yes | NA | Teams App ID of the underlying app which will be used to power the content sharing. Required if allow-share-in-meeting is set as true. |
+| entityName | `data-entity-name` | String | Optional | NA | App entity name. |
+| entityDescription | `data-entity-description` | String | Optional | NA | Description of app content to share. |
+| locale | `data-locale` | String | Optional | en-US | User preferred language. |
+| target | `data-target` | String | Optional | self | Specifies whether the link opens in the same window, new tab, or new window. |
+| buttonType | `data-button-type` | String | No | primaryShareInMeeting | Specifies the button background color: `primaryShareInMeeting` or `secondaryShareInMeeting`. |
+| buttonSize | `data-button-size` | String | No | NA | Button size in pixels. |
+| shareinmeeting | `data-allow-share-in-meeting` | String | No | NA | Button size in pixels. |
+
+## End user experience
+
+# [Chat or channel](#tab/chatchannel)
+
+To share the content to a chat or channel:
+
+1. Open the web app in the browser and select **Share to Teams**. The Share to Teams dialog opens.
+
+    :::image type="content" source="../../assets/images/share-to-teams/share-to-teams-browser.png" alt-text="Screenshot shows the Share to Teams button in the browser.":::
+
+1. Add the chat or channel name.
+
+    :::image type="content" source="~/assets/images/share-to-teams-popup.png" alt-text="Screenshot shows the Share to Teams dialog to add chat or channel name.":::
+
+   > [!NOTE]
+   > If the app isn't added in chat or channel scope, an app consent appears in the Share to Teams dialog.
+
+1. Select **Share**. The app gets added in the shared scope.
+
+# [Live meeting](#tab/live)
+
+To share content to an ongoing meeting:
+
+1. Open the web app in the browser and select **Share to Teams**. The Share to Teams dialog opens.
 
-1. If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force **Share** to render at the appropriate place in the pipeline.
+    :::image type="content" source="../../assets/images/share-to-teams/share-to-teams-browser.png" alt-text="Screenshot shows the Share to Teams button in the browser to share in meeting.":::
+
+1. Add the meeting name.
```

Comment:
To share in the ongoing meeting, user can directly click on the present now button. They will not need to add the meeting name.